src/server/pkg/metrics/segment.go

```diff
@@ -0,0 +1,27 @@
+package metrics
+
+import (
+	"github.com/segmentio/analytics-go"
+	"go.pedge.io/lion"
+)
+
+func reportSegment(metrics *Metrics) {
+	client := analytics.New("hhxbyr7x50w3jtgcwcZUyOFrTf4VNMrD")
```

Comment:
I wondered about that myself but I think this is actually a reasonable thing to do. Segment keys aren't actually a security issue since they're write only, it's pretty common for people to just embed them in their websites javascript. Putting them in a secret wouldn't do much since we'd be shipping the secret as part of our manifest, which wouldn't keep it secret from anyone.